controllers/feeds/getFeeds.js

```diff
@@ -107,7 +107,7 @@ module.exports = async (req, res) => {
               req.userId
             );
             let newItem = await modifyAnonimityPost(item, isBlurredPost);
-            newItem = modifyReactionsPost(newItem, newItem.anonimity);
+            newItem = modifyReactionsPost(newItem);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `modifyReactionsPost` function has been updated to no longer require the `anonimity` parameter. This change could potentially introduce bugs if the `modifyReactionsPost` function uses the `anonimity` property internally and expects it as an argument. Please ensure that the `modifyReactionsPost` function is updated accordingly and does not rely on this parameter, or consider passing the `newItem.anonimity` directly from the `newItem` object if necessary.
```diff
- newItem = modifyReactionsPost(newItem);
+ newItem = modifyReactionsPost(newItem, newItem.anonimity);
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->